templates/Web-deployment.yaml

```diff
@@ -23,6 +23,24 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-web
         app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-web
     spec:
+      nodeSelector:
```

Comment:
Hi @adamcharnock!

Could you remove the lines 26 to 31?
The goal is to leave only the lines 32 to 43 that will be filled in only when someone sets the custom values for these properties.